src/System.Private.CoreLib/src/System/Environment.cs

```diff
@@ -59,6 +59,7 @@ public static void FailFast(String message, Exception exception)
             RuntimeExceptionHelpers.FailFast(message, exception);
         }
 
+#if !PLATFORM_UNIX
```

Comment:
Could you please move this to `Environment.Windows.cs` instead of adding the ifdef?

(BTW: We are using corefx coding guidelines in CoreRT - https://github.com/dotnet/corefx/tree/master/Documentation/coding-guidelines.)